packages/viz/src/lib/style/helpers/color-bins-style.ts

```diff
@@ -5,27 +5,55 @@ import {
   findIndexForBinBuckets
 } from './utils';
 
-import { Classifier } from '../../utils/Classifier';
+import { Classifier, ClassificationMethod } from '../../utils/Classifier';
 import { Style } from '../Style';
 import {
   CartoStylingError,
   stylingErrorTypes
 } from '../../errors/styling-error';
-import { ColorBinsStyleOptions, defaultColorBinsStyleOptions } from '..';
 import { StyledLayer } from '../layer-style';
-import { toDeckStyles } from './style-transform';
 import { NumericFieldStats, GeometryType } from '../../sources/Source';
+import { getStyleValue, BasicOptionsStyle, getStyles } from '../default-styles';
+
+export interface ColorBinsOptionsStyle extends Partial<BasicOptionsStyle> {
```

Comment:
Does it mean we will be able to 'override' any style property, for example set the size in the helper itself?

Reply:
Yes! In the previous version, if you were using the library in typescript you had to do:

```javascript
const userOptions = { palette: 'teal'}
const options = { ...defaultColorBinsOptionsStyle, ...userOptions}
const style = carto.viz.colorBinsStyle('temp', options)
```

Now you can just do
```javascript
const style = carto.viz.colorBinsStyle('temp', { palette: 'teal'})
```